clients/fides-js/src/lib/initialize.ts

```diff
@@ -270,7 +271,9 @@ export const getInitialFides = ({
     }) => PrivacyExperience;
   }): Partial<FidesGlobal> | null => {
   const hasExistingCookie = !isNewFidesCookie(cookie);
-  if (!hasExistingCookie && !options.fidesString) {
+  const otConsentCookie = getOTConsentCookie();
+  const isOtMigrationMode = !!otConsentCookie && !!options.otFidesMapping;
+  if (!hasExistingCookie && !options.fidesString && !isOtMigrationMode) {
```

Comment:
For OT Migration mode, we need to ensure that we proceed in this fn to update the experience with cookie consent (at this point the cookie now holds those new OT consent overrides)